packages/lesswrong/components/review/ReviewVoteTableRow.tsx

```diff
@@ -177,14 +190,15 @@ const arrayDiff = (arr1:Array<any>, arr2:Array<any>) => {
 }
 
 const ReviewVoteTableRow = (
-  { post, dispatch, costTotal, classes, expandedPostId, currentVote, showKarmaVotes }: {
+  { post, dispatch, costTotal, classes, expandedPostId, currentVote, showKarmaVotes, reviewPhase }: {
     post: PostsListWithVotes,
     costTotal?: number,
     dispatch: React.Dispatch<SyntheticQualitativeVote>,
     showKarmaVotes: boolean,
     classes:ClassesType,
     expandedPostId?: string|null,
     currentVote: SyntheticQualitativeVote|null,
+    reviewPhase?: ReviewPhase | void
```

Comment:
I don't think we want to have `| void`, but if this is coming from the incomplete function definition it'll get changed later.  Probably `| undefined`?